### PR TITLE
animated aleph eye

### PR DESCRIPTION
**Pull request by triggers0**
animated aleph eye

### CODE REVIEW

**Comment by triggers0** [comment]
[Creepy eyes following cursor on landing page](https://trello.com/c/pxWq6kfj/46-creepy-eyes-following-cursor-on-landing-page)

File: frontend/src/main/scala/org/kys/athena/App.scala [hunk 1]
```diff
@@ -50,18 +52,17 @@ object App {
...
+        LandingPage.render(windowEvents.onMouseMove)
       }.collectStatic(RouteNotFound) {
-      LandingPage.render
+      LandingPage.render(windowEvents.onMouseMove)
     }.collect[OngoingRoute] { page =>
       OngoingPage.render(page, hideSearchBar.writer)
     }
 
-  def render(): HtmlElement = {
+    def render(): HtmlElement = {
```
Comment (greg2010) [inline frontend/src/main/scala/org/kys/athena/App.scala:65]: fmt

File: frontend/src/main/scala/org/kys/athena/components/LandingPage.scala [hunk 2]
```diff
@@ -2,11 +2,69 @@ package org.kys.athena.components
...
-  def render: HtmlElement = {
+
+
+  def render(mouseES:EventStream[MouseEvent]): HtmlElement = {
+    val referenceDiv = div(
+      position := "relative",
+      top := "44.5%",
+      left := "56%",
+      width := "0px",
+      height := "0px",
```
Comment (greg2010) [inline frontend/src/main/scala/org/kys/athena/components/LandingPage.scala:18]: fmt

File: frontend/src/main/scala/org/kys/athena/components/LandingPage.scala [hunk 3]
```diff
@@ -2,11 +2,69 @@ package org.kys.athena.components
...
+      )
+
+    val eyecoords = mouseES.map{ev=>
```
Comment (greg2010) [inline frontend/src/main/scala/org/kys/athena/components/LandingPage.scala:21]: fmt

File: frontend/src/main/scala/org/kys/athena/components/LandingPage.scala [hunk 4]
```diff
@@ -2,11 +2,69 @@ package org.kys.athena.components
...
+      val d = sqrt(pow(ev.pageX-eyeRef.left, 2) + pow(ev.pageY-eyeRef.top, 2))
+      // calc theta from mouse coordinates
+      val th = atan2(ev.pageY-eyeRef.top, ev.pageX-eyeRef.left)
+      // determine distance % mouse is from center and then multiply by polar distance coord "r" (tan-1 (y/x) multiplied by max allowed circle draw
+      val r = sqrt(pow(1, 2) + pow(5, 2))
+      //val maxd = //determined from user screen?
+      //val maxr = //determined from aleph pic
+      // convert back to cartesian
+      //val x = (maxr * (d / maxd) * cos(th))
+      //val y = (maxr * (d / maxd) * sin(th))
```
Comment (greg2010) [inline frontend/src/main/scala/org/kys/athena/components/LandingPage.scala:32]: Remove unnecessary comments

**Review by greg2010 — COMMENTED** [review]
Needs a linting pass, otherwise LGTM

**Pull request merged**